src/screens/Signup/index.tsx

```diff
@@ -13,24 +17,26 @@ import {
 import logo from './assets/wolox-logo.png';
 import styles from './styles.module.scss';
 
-interface SignupForm {
-  firstName: string;
-  lastName: string;
-  email: string;
-  password: string;
-  passwordConfirmation: string;
-}
-
 function Signup() {
   const { t, i18n } = useTranslation();
+
   const {
     register,
     handleSubmit,
     watch,
     formState: { errors }
-  } = useForm<SignupForm>();
+  } = useForm<User>();
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [state, loading, error, sendRequest] = useLazyRequest({ request: signUp });
```

Comment:
There's no need to disable this rule, only to match the destructured props following it's pattern. For example:
```TSX
  const [, loading, , sendRequest] = useLazyRequest({ request: signUp });
```